scripts/systems.py

```diff
@@ -87,7 +87,7 @@
 # Env
 export MPICH_MAX_THREAD_SAFETY=multiple
 export MIMALLOC_EAGER_COMMIT_DELAY=0
-export MIMALLOC_LARGE_OS_PAGES=1
+export MIMALLOC_ALLOW_LARGE_OS_PAGES=1
```

Comment:
In the generator scripts we had 64 for `daint-mc` (which corresponds to the default value) and 128 for `daint-gpu`.

I'm ok leaving it out for `daint-mc` (even if it does not harm specifying it), but for `daint-gpu` I think it is missing.

```suggestion
export MIMALLOC_ALLOW_LARGE_OS_PAGES=1
export DLAF_BT_BAND_TO_TRIDIAG_HH_APPLY_GROUP_SIZE=128
```

I know that Piz Daint is going to be removed sooner rather than later, it's absolutely a minor comment.